### PR TITLE
[36] Added photos to groups

### PR DESCRIPTION
**Pull request by Szyntos**
Added photos to groups and a mutation for randomly assigning unique picture.

Assigning photos is done by mutation:

```graphql
mutation MyMutation {
  assignPhotosToGroups(editionId: 1)
}
```
Pictures that are to be used  for groups should have a FileType = "image/group"

FileType from now on will mean the usecase of the file

### CODE REVIEW

File: backend/resources/files/gr9.png
Comment (Jullija) [inline backend/resources/files/gr9.png:1]: ja z ziomkami, gdy piwo

File: backend/resources/files/gr7.png
Comment (Jullija) [inline backend/resources/files/gr7.png:1]: ty z mamą i mamą

File: backend/resources/files/gr5.png
Comment (Jullija) [inline backend/resources/files/gr5.png:1]: żugajki gdy Julia Żugaj

File: backend/src/main/kotlin/backend/groups/Groups.kt [hunk 1]
```diff
@@ -35,7 +36,11 @@ class Groups(
 
     @ManyToOne(fetch = FetchType.LAZY)
     @JoinColumn(name = "edition_id", nullable = false)
-    var edition: Edition
+    var edition: Edition,
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "image_file_id")
```
Comment (Jullija) [inline backend/src/main/kotlin/backend/groups/Groups.kt:42]: maybe add `nullable=true`

File: backend/src/main/python/insert_data.py [hunk 2]
```diff
@@ -50,10 +50,17 @@ def insert_data(data_count_multiplier=1):
     owlbear_filenames = ["owlbear1.png", "owlbear2.png", "owlbear3.png", "owlbear4.png",
                          "owlbear5.png", "owlbear6.png", "owlbear7.png"]
 
+    group_filenames = [f"gr{i}.png" for i in range(1, 21)]
```
Comment (Jullija) [inline backend/src/main/python/insert_data.py:53]: maybe gr_1, gr_2, ... instead of gr1, gr2, .... But it's just a random thought

**Review by Jullija — APPROVED** [review]
200w

File: backend/src/main/kotlin/backend/groups/Groups.kt (hunk 1 above)
Comment (Szyntos) [reply backend/src/main/kotlin/backend/groups/Groups.kt:42]: yup, it's in the db, but not here, will add

Comment (Szyntos) [reply backend/src/main/kotlin/backend/groups/Groups.kt:42]: all fields are nullable by default, sooo

**Pull request merged**